modules/library/main/src/main/java/org/geotools/styling/Normalize.java

```diff
@@ -0,0 +1,49 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2015, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.styling;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.opengis.filter.expression.Expression;
+import org.opengis.style.ContrastMethod;
+
+/**
+ * @author iant
+ *
+ */
+public class Normalize extends AbstractContrastEnhancementMethod{
+
+    final static List<String> ALGORITHM_NAMES = Arrays.asList("StretchToMinimumMaximum",
+            "ClipToMinimumMaximum", "ClipToZero");
```

Comment:
Document as static constants so we can provide javadocs, and so developers do not have to type this stuff in.